restore-cognito.md

```diff
@@ -0,0 +1,32 @@
+Prerequisites 
+For creating and executing backups, you can use the npm package called `cognito-backup-restore` 
+(do `npm install cognito-backup-restore` to get the package)
+
+Required IAM permissions
+If you want to create a backup of the users you will need the permission - `ListUsers`
+Restoring the users requires - `AdminCreateUser`
+There might be more roles required to do all the operations for getting and restoring users. If you need a permission added
+speak with Iva or Pol, who will add it through IAM.
```

Comment:
this needs to be tested now and figured out. If we are in a situation when we need to restore users in Cognito, the situation will surely be bad enough and we will be in a rush. We need a runbook that is tested and proven to work -- we will be in a stressful enough situation to have to also work such things out in addition